src/decode.c

```diff
@@ -781,6 +801,19 @@ void DecodeGlobalConfig(void)
             decoder_max_layers = value;
         }
     }
+    GetPacketMaxAlertConfig();
+}
+
+void GetPacketMaxAlertConfig(void)
+{
+    intmax_t max = 0;
+    if (ConfGetInt("packet-alert-max", &max) == 1) {
+        packet_alert_max = max;
+        BUG_ON(max <= 0);
```

Comment:
Q: Why don't we fall back to defaults in a case like this?

Reply:
Instead of having `BUG_ON`? On the one hand, I can see why this would be better, for not disrupting anything. On the other, I wonder if it is not better to make the person aware that whatever setting they wanted to use went wrong somehow? Because if we reach this case it means the person tried to change the `PACKET_ALERT_MAX` config, but used an invalid value... -- In which case maybe we should have an error message. Hm... I need more input on this. Gonna ask Victor.